.github/workflows/ccpp.yml

```diff
@@ -19,3 +19,42 @@ jobs:
       run: make -C demo/glfw_opengl3
     - name: build example
       run: make -C example
+
+  build_msvc:
+    strategy:
+      matrix:
+        arch: [x86, x64]
+        define: ['', '-DINCLUDE_ALL']
+    runs-on: windows-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Visual Studio shell
+      uses: egor-tensin/vs-shell@v2
```

Comment:
Please do not merge it yet, I should replace this with `ilammy/msvc-dev-cmd` instead, since that is more actively used / developed

Reply:
Take your time. Let me know once we should jump on this to review :wink:. Thanks!

Reply:
@dumblob I have updated the builder, so feel free to review it.

Reply:
`apt-get` seems to have some trouble ...